src/components/OrderConfirmation/index.tsx

```diff
@@ -102,7 +103,11 @@ function OrderConfirmation({ title, pathname, currentAmount }: IOrderConfirmatio
               <Box flex="1">
                 <Typography className={classes.payLabel}>{title}</Typography>
               </Box>
-              <Button className={classes.orderTotalBtn} onClick={() => setDisplayKeypad(true)}>
+              <Button
+                data-testid="orderTotalButton"
```

Comment:
Just to keep this consistent with the other attribute names, could you change this please to:
```suggestion
                data-test-id="orderTotalButton"
```

Reply:
Hi @mattgresham - The react testing library method getByTestId only supports `data-testid`. I tried `data-test-id` earlier but the selector doesn't work. See docs https://testing-library.com/docs/queries/bytestid/